testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -0,0 +1,83 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import print_function
+import numpy as np
+import MDAnalysis
+import MDAnalysis.analysis.pca as pca
+from MDAnalysis.analysis.align import _fit_to
+
+from numpy.testing import (assert_almost_equal, assert_equal,
+                           assert_array_almost_equal, raises)
+
+from MDAnalysisTests.datafiles import (PDB, XTC, RANDOM_WALK, RANDOM_WALK_TOPO,
+                                       waterPSF, waterDCD)
+
+
+class TestPCA(object):
+    def setUp(self):
+        self.u = MDAnalysis.Universe(PDB, XTC)
+        self.pca = pca.PCA(self.u.atoms, select='backbone and name CA',
+                           align=False)
+        self.pca.run()
+        self.n_atoms = self.u.select_atoms('backbone and name CA').n_atoms
+
+    def test_cov(self):
+        atoms = self.u.select_atoms('backbone and name CA')
+        xyz = np.zeros((10, 642))
```

Comment:
to make this more resilient for change use `np.zeros((self.u.trajectory.n_frames, self.n_atoms * 3))`